mipidsi/README.md

```diff
@@ -0,0 +1,65 @@
+# mipidsi
```

Comment:
I assume this readme is unchanged? The GitHub interface didn't show a diff because the file has been moved.